suite-common/metadata-types/src/metadataTypes.ts

```diff
@@ -239,3 +239,6 @@ export interface MetadataState {
 
 export type OAuthServerEnvironment = 'production' | 'staging' | 'localhost';
 export type MetadataEncryptionVersion = 1 | 2;
+
+// todo:
```

Comment:
Maybe something forgotten or unfinished?

Reply:
unfinished. I don't care at this stage. we are not sure if this feature will ever land "production" or not 

anyway this the shape
![image](https://github.com/trezor/trezor-suite/assets/30367552/ac6c3e82-a41b-4c7d-be5e-d793aeaa68c1)

Reply:
Ok, got it. So maybe we can remove the todo at all or extend it to something like:

`// Todo: structure of the Password Entry not finalized yet.`